widget/menu.go

```diff
@@ -18,6 +18,66 @@ type Menu struct {
 	OnDismiss   func()
 	activeItem  *menuItem
 	customSized bool
+	index       int
+}
+
+func (m *Menu) Defocus() {
+	for _,o := range m.Items {
+		if mi, ok := o.(*menuItem); ok {
+			mi.FocusLost()
+		}
+	}
+}
+
+func (m *Menu) Do() {
```

Comment:
What does "Do" mean?

Reply:
Do is called when the Enter key is pressed, and will simulate tapping. The comments are updated now.